decidim-core/app/views/layouts/decidim/_application.html.erb

```diff
@@ -15,6 +15,7 @@
   <head>
     <title><%= decidim_page_title %></title>
     <%= render partial: "layouts/decidim/head" %>
+    <%= render partial: "layouts/decidim/js_configuration" %>
```

Comment:
I needed to move this to the head tag because when embedding an iframe it wasn't being executed, and I couldn't find the reason why. Let me know if you want me to get deeper on the reasons